ext/intl/dateformat/datepatterngenerator_class.h

```diff
@@ -0,0 +1,69 @@
+/*
+   +----------------------------------------------------------------------+
+   | This source file is subject to version 3.01 of the PHP license,      |
+   | that is bundled with this package in the file LICENSE, and is        |
+   | available through the world-wide-web at the following url:           |
+   | http://www.php.net/license/3_01.txt                                  |
+   | If you did not receive a copy of the PHP license and are unable to   |
+   | obtain it through the world-wide-web, please send a note to          |
+   | license@php.net so we can mail you a copy immediately.               |
+   +----------------------------------------------------------------------+
+   | Authors: Mel Dafert (mel@dafert.at)                                  |
+   +----------------------------------------------------------------------+
+*/
+
+#ifndef DATEPATTERNGENERATOR_CLASS_H
+#define DATEPATTERNGENERATOR_CLASS_H
+
+//redefinition of inline in PHP headers causes problems, so include this before
+#include <math.h>
+
+#include <php.h>
+#include "intl_common.h"
+#include "intl_error.h"
+#include "intl_data.h"
+
+#ifndef USE_DATETIMEPATTERNGENERATOR_POINTER
+typedef void DateTimePatternGenerator;
+#else
+using icu::DateTimePatternGenerator;
+#endif
+
+typedef struct {
+	// 	error handling
+	intl_error  err;
+
+	// ICU break iterator
+	DateTimePatternGenerator*	dtpg;
```

Comment:
```suggestion
	DateTimePatternGenerator *dtpg;
```
Avoid tabs for alignment -- and this doesn't actually align with anything...